ptf/sainexthopgroup.py

```diff
@@ -3731,3 +3731,515 @@ def l3IPv6EcmpSVIPortLagHostTest(self):
         finally:
             sai_thrift_remove_fdb_entry(self.client, fdb_entry1)
             sai_thrift_remove_fdb_entry(self.client, fdb_entry2)
+
+
+@group("draft")
```

Comment:
please remove this tag for marking it as checked and runnabled

Reply:
done